command/src/proxy.rs

```diff
@@ -551,8 +551,10 @@ impl FromStr for LoadBalancingAlgorithms {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
```

Comment:
Could you update the string to disable case sensitive check like this:
```suggestion
        match s.to_lowercase().as_str() {
```